### ISSUE TITLE
Improved Documentation

### ISSUE THREAD

**Issue by matJC** [juliangruber/merge-pull-request-action#20]
This action functionally works great but could use better documentation to make it more accessible for people.

Thanks for the great work

**Comment by juliangruber** [comment]
Hey, could you suggest what can be added to the documentation?

**Comment by hugoheml** [comment]
Hey,
I think you can add the `method` property values list in the documentation

**Comment by juliangruber** [comment]
good idea! https://github.com/juliangruber/merge-pull-request-action/commit/d49537f415bda5a1afa45a9df2f4f07c1e9d22bd
